packages/gestalt/src/icons/heart-outline.svg

```diff
@@ -0,0 +1 @@
+<svg height="24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="M17.76 3a6.263 6.263 0 0 0-4.416 1.809L12 6.154l-1.344-1.345A6.265 6.265 0 0 0 6.239 3a6.262 6.262 0 0 0-4.417 1.809 6.193 6.193 0 0 0 0 8.774l9.13 8.991c.292.284.67.426 1.048.426s.755-.142 1.047-.426l9.13-8.991a6.193 6.193 0 0 0 0-8.774A6.263 6.263 0 0 0 17.762 3m0 2c1.139 0 2.208.437 3.005 1.226.796.793 1.234 1.848 1.234 2.97s-.438 2.177-1.225 2.962L12 20.798l-8.766-8.632C2.438 11.373 2 10.318 2 9.196s.438-2.177 1.228-2.965A4.253 4.253 0 0 1 6.238 5c1.139 0 2.208.437 3.003 1.224l1.344 1.344L12 8.983l1.414-1.415 1.337-1.337A4.249 4.249 0 0 1 17.76 5" fill="#8e8e8e"/></svg>
```

Comment:
@qywang looks like these 2 assets can be optimized some more, can we check with PDS team why they are not the minified ones ready for gestalt?
![image](https://user-images.githubusercontent.com/7877010/58209068-1ffd9b80-7c9b-11e9-92ba-cd055ba0febd.png)

Reply:
Hmmm I checked with them before pushing the change. Let me ask again. Thanks!